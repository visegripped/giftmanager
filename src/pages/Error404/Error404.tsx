import './Error404.css';

const Error404 = () => {
  return (
    <>
      <section>
        <h2>404 - this page can not be found.</h2>
      </section>
    </>
  );
};

export default Error404;
