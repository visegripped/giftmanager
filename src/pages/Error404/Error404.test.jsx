import Error404 from './Error404';
import { render } from '@testing-library/react';

describe('The Error404 page', () => {
  describe('when no user context is provided', () => {
    it('should match snapshot', () => {
      const { container } = render(<Error404 />);
      expect(container).toMatchSnapshot();
    });
  });
});
