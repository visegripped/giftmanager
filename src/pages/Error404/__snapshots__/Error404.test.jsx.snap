// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`The Error404 page > when no user context is provided > should match snapshot 1`] = `
<div>
  <section>
    <h2>
      404 - this page can not be found.
    </h2>
  </section>
</div>
`;
